docs/tasks.md

```diff
@@ -52,6 +52,7 @@ The following tables give you an overview of the tasks in MTEB.
 | [BlurbsClusteringS2S.v2](https://www.inf.uni-hamburg.de/en/inst/ab/lt/resources/data/germeval-2019-hmc.html) (Steffen Remus, 2019) | ['deu'] | Clustering | s2s | [Fiction, Written] | {'test': 2048} | {'test': 23.02} |
 | [BornholmBitextMining](https://aclanthology.org/W19-6138/) | ['dan'] | BitextMining | s2s | [Web, Social, Fiction, Written] | {'test': 500} | {'test': 89.7} |
 | [BrazilianToxicTweetsClassification](https://paperswithcode.com/dataset/told-br) (Joao Augusto Leite and                         Diego F. Silva and                         Kalina Bontcheva and                         Carolina Scarton, 2020) | ['por'] | MultilabelClassification | s2s | [Constructed, Written] | {'test': 2048} | {'test': 85.05} |
+| [BrightRetrieval](https://huggingface.co/datasets/xlangai/BRIGHT) (Su et al., 2024) | ['eng'] | Retrieval | s2p | [Non-fiction] | {'standard': 1334914, 'long': 7048} | {'standard': 800.3994729248476, 'long': 46527.35839954597} |
```

Comment:
This does not seem relevant to the PR. Wondering if this would go away if we merge updates from `main`?

Reply:
@isaac-chung As you can see from the commit logs, it is done by the github actions. It comes back even if I deleted it.